### ISSUE TITLE
Drop-down in table header inherits styling

### ISSUE THREAD

**Issue by 3af** [react-toolbox/react-toolbox#1799]
I think the drop-down should look the same no matter what, right?

Regular drop-down:
![obrazek](https://user-images.githubusercontent.com/33279089/35678723-ba141c7a-0754-11e8-8c94-b0ad3be94bf8.png)

Drop-down in table header:
![obrazek](https://user-images.githubusercontent.com/33279089/35678797-f9215b30-0754-11e8-8d2d-57418f2527df.png)

[Repository for reproduction.](https://github.com/3af/react-toolbox-table-dropdown-collision)

I'll open PR fixing this soon. I will set the font color to [black](https://github.com/react-toolbox/react-toolbox/blob/a841d2a9c4c398281101feb687bc65f8cdd37902/components/colors.css#L275) and size to [16px](https://github.com/react-toolbox/react-toolbox/blob/a841d2a9c4c398281101feb687bc65f8cdd37902/components/variables.css#L17).

**Comment by rubenmoya** [comment]
Thank you for reporting this!

It should follow [the spec](https://material.io/guidelines/components/data-tables.html#data-tables-interaction), I'm not sure if it's the same as the dropdown but it's worth to have a look.

**Comment by 3af** [comment]
I think drop-down is a specialized [menu](https://material.io/guidelines/components/menus.html#menus-specs) and the spec you posted suggests

```
Inline menu
    13sp Roboto Regular
    87% black
    Follow spacing and sizing guidelines for menus
```

and it seems that 13sp is 12px and I'll use the 87% black. Does that seems correct for you @rubenmoya?

**Comment by rubenmoya** [comment]
Yeah, sounds good! I'm pretty `87% black` is `--palette-grey-900`, which is also our `--color-text`.

**Comment by 3af** [comment]
I was researching this and it seems that since RT uses CSS modules I can't fix this by changing [CSS of dropdown](https://github.com/react-toolbox/react-toolbox/blob/a841d2a9c4c398281101feb687bc65f8cdd37902/components/dropdown/theme.css#L128) like
```
.table .values {
  color: var(--color-text);
  font-size: var(--font-size-tiny);
```
because `.table` comes from `Table` CSS module and will be processed by webpack.

The solution seems to be creating new component called something like `TableDropdown` which will set appropriate CSS class which will be defined in dropdown theme. Then it's up to the developer to use this component inside `Table`.

Can anyone confirm whether this is correct approach? Maybe there is other way. I'm still unexperienced with CSS modules. Thanks.
